models/graph.py

```diff
@@ -34,6 +35,11 @@
 
 
 def select_default_color(item_field):
+    """
+    returns color for lines

```

Comment:
Als commando schrijven: "Return color for lines" (dus niet 'returnS')

(en raar genoeg was volgens mij de google stijl voor dat `:return:` juist `:returns:`, maar dat doe ik uit m'n geheugen.